src/notation/inotationconfiguration.h

```diff
@@ -194,6 +194,14 @@ class INotationConfiguration : MODULE_EXPORT_INTERFACE
 
     virtual io::path_t styleFileImportPath() const = 0;
     virtual void setStyleFileImportPath(const io::path_t& path) = 0;
+
+    virtual int lastPageRow() const = 0;
+    virtual void setLastPageRow(int value) = 0;
+
+    virtual int lastSubPageRow() const = 0;
+    virtual void setLastSubPageRow(int value) = 0;
+
+    virtual void resetStyleIndex() = 0;
```

Comment:
Now that these are in INotationConfiguration, which is quite a widely-used class, their names are a bit too general. I'd recommend making them a bit more specific:
- `styleDialogLastPageIndex()` (and the corresponding setter and related things in `internal/notationconfiguration.h`)
- `styleDialogLastSubPageIndex()` (idem)
- `resetStyleDialogPageIndices()`

That way, it's immediately clear that these methods are about the style dialog, and not about who-knows-what-else.